starter/slack-spring-boot/src/main/kotlin/io/olaph/slack/broker/security/VerificationMethodArgumentResolver.kt

```diff
@@ -73,4 +76,17 @@ abstract class VerificationMethodArgumentResolver(private val signingSecret: Str
         }
     }
 
+    /**
+     * since spring decodes unreserved characters automatically
+     * we need to replace them with their url escaped value again
+     * http://www.rfc-editor.org/rfc/rfc1738.txt
+     */
+    private fun replaceSpecialChars(payload: String): String {
```

Comment:
this does not need to be an instance method, this should be in the companion object